Kernel/Devices/FloppyDiskDevice.cpp

```diff
@@ -230,6 +235,7 @@ bool FloppyDiskDevice::write_sectors_with_dma(u16 lba, u16 count, const u8* inbu
     u16 cylinder = lba2cylinder(lba);
     u16 head = lba2head(lba);
     u16 sector = lba2sector(lba);
+    u8 drivesel = (is_slave() == true) ? 1 : 0;
```

Comment:
This seems excessively verbose.
```suggestion
    u8 drivesel = is_slave();
```